### PR TITLE
feat: Add operator id in `operator keys list`

### PR DESCRIPTION
**Pull request by fyInALT**
### What Changed?

When use `operator keys list `, can see the operator id for bls key:

```bash
go run ./cmd/eigenlayer/main.go  operator keys list             
Key Name: holeskyt1
Key Type: BLS
Public Key: E([498211989701534593628498974128726712526336918939770789545660245177948853517,19434346619705907282579203143605058653932187676054178921788041096426532277474])
Operator Id: 0x5a76fe9014f9cd296a69ac589c2bbd2c6a354c5e4c0c79ee35c5b8202b8523a2
Key location: /home/cc/.eigenlayer/operator_keys/holeskyt1.bls.key.json
```

Add `Operator Id`.

### CODE REVIEW

File: pkg/operator/keys/list.go [hunk 1]
```diff
@@ -91,6 +101,53 @@ func GetPubKey(keyStoreFile string) (string, error) {
 	}
 }
 
+func GetOperatorIdFromPubKey(pubKey string) (string, error) {
+	// From:
```
Comment (NimaVaziri) [inline pkg/operator/keys/list.go:105]: @fyInALT was this meant to be included?

Comment (fyInALT) [reply pkg/operator/keys/list.go]: Not need include, i just change comments more clear.

**Comment by shrimalmadhur** [comment]
@fyInALT you might want to re-use the eigensdk utility from here - https://github.com/Layr-Labs/eigensdk-go/blob/master/types/operator.go#L118

**Comment by shrimalmadhur** [comment]
> > @fyInALT you might want to re-use the eigensdk utility from here - https://github.com/Layr-Labs/eigensdk-go/blob/master/types/operator.go#L118
> 
> The problem is if we not use password(in `list` cmd, we not need depend user enter password), we can not got `bls.KeyPair` directly, because it contain the prikey, so in this case we cannot re-use the eigensdk 's code, we also can add a utility interface in eigensdk, but i think this func maybe not used in other env.

good point. @samlaf maybe sdk should also support this - operator id just from pub key and not from keypair. Not sure if it's needed in sdk - if you think it's needed, we should add this in sdk otherwise keep it here? thoughts?

**Comment by samlaf** [comment]
> > > @fyInALT you might want to re-use the eigensdk utility from here - https://github.com/Layr-Labs/eigensdk-go/blob/master/types/operator.go#L118
> > 
> > 
> > The problem is if we not use password(in `list` cmd, we not need depend user enter password), we can not got `bls.KeyPair` directly, because it contain the prikey, so in this case we cannot re-use the eigensdk 's code, we also can add a utility interface in eigensdk, but i think this func maybe not used in other env.
> 
> good point. @samlaf maybe sdk should also support this - operator id just from pub key and not from keypair. Not sure if it's needed in sdk - if you think it's needed, we should add this in sdk otherwise keep it here? thoughts?

https://github.com/Layr-Labs/eigensdk-go/blob/4a204d0e0c9118babf1d74353f72c2b97009d7db/types/operator.go#L114

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**